downstream/modules/platform/ref-controller-inventory-sync-details.adoc

```diff
@@ -0,0 +1,27 @@
+[id="controller-inventory-sync-details"]
+
+= Inventory sync details
+
+Access the *Details* tab to view details about the job execution:
+
+image::ug-show-job-details-for-inv-sync.png[Show job details for inventory sync]
+
+You can view the following details for an executed job:
+
+Status:: It can be any of the following:
+* *Pending*: The inventory sync has been created, but not queued or started yet. 
+Any job, not just inventory source syncs, stay in pending until it is ready to be run by the system. 
+Reasons for inventory source syncs not being ready include dependencies that are currently running (all dependencies must be completed before the next step can execute), or there is not enough capacity to run in the locations it is configured to.
```

Comment:
I had a similar thing.

Reasons for inventory source syncs not being ready include:

* Dependencies that are currently running (all dependencies must be completed before the next step can execute).
* Insufficient capacity to run in the locations it is configured for.